projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/interestrate/future/provider/InterestRateFutureOptionMarginSecurityNormalSmileMethod.java

```diff
@@ -161,4 +168,143 @@ public double underlyingFuturesPrice(final InterestRateFutureOptionMarginSecurit
     return METHOD_FUTURE.price(security.getUnderlyingFuture(), multicurves);
   }
 
+  /**
+   * Computes the option security price delta, wrt the futures price dV/df. The futures price is computed without convexity adjustment.
+   * It is supposed that for a given strike the volatility does not change with the curves.
+   * @param security The future option security.
+   * @param normalData The curve and normal volatility data.
+   * @return The delta.
+   */
+  public double priceDelta(InterestRateFutureOptionMarginSecurity security,
+      NormalSTIRFuturesProviderInterface normalData) {
+    ArgumentChecker.notNull(security, "Option security");
+    ArgumentChecker.notNull(normalData, "Normal data");
+    double priceFuture = METHOD_FUTURE.price(security.getUnderlyingFuture(), normalData.getMulticurveProvider());
+    return priceDeltaFromFuturePrice(security, normalData, priceFuture);
+  }
+
+  /**
+   * Computes the option security price delta, wrt the futures price dV/df. The futures price is computed without convexity adjustment.
+   * It is supposed that for a given strike the volatility does not change with the curves.
```

Comment:
As the futures price is passed directly, saying that the "price is computed without convexity adjustment" does not make sense here.